gpkitmodels/GP/aircraft/tail/tail_tests.py

```diff
@@ -50,17 +51,62 @@ def test_emp():
     vtperf = emp.vtail.flight_model(emp.vtail, fs)
     tbperf = emp.tailboom.flight_model(emp.tailboom, fs)
 
+
+    m = Model(htperf.Cd + vtperf.Cd + tbperf.Cf,
+              [emp.vtail.lv == emp.tailboom.l, emp.htail.lh == emp.tailboom.l,
+               emp.htail.Vh <= emp.htail.planform.S*emp.htail.lh/Sw/cmac,
+               emp.vtail.Vv <= emp.vtail.planform.S*emp.vtail.lv/Sw/bw,
+               emp, fs, htperf, vtperf, tbperf])
+
+    from gpkit import settings
+    if settings["default_solver"] == "cvxopt":
```

Comment:
iiiiinteresting

Reply:
what is interesting?

Reply:
we've got a collection of cvxopt hacks scattered around the gpkit tests as well; maybe at some point we can abstract them and integrate these observations into error messages / documentation?

Reply:
Yeah that's not a bad idea